src/main/java/com/uid2/shared/middleware/AuthMiddleware.java

```diff
@@ -198,6 +214,24 @@ public void handle(RoutingContext rc) {
             }
         }
 
+        public void handleV2(RoutingContext rc, Role role) {
+            // add aws request id tracer to help validation
```

Comment:
This looks like a copy paste from above and does not really belong to auth logic. Can this be extracted into a re-usable method or, even better, into a handler of its own?

Reply:
Reverted. Thought v2 auth may be different.